pages/pipelines/pipelines.md.erb

```diff
@@ -62,4 +62,35 @@ Your pipeline’s name, description, its repository, and your default branch are
 
 The display settings can be found in the `Pipeline Settings` section. Adding a description for your pipeline is optional, but name, repository, and default branch are all required. Descriptions have full emoji support! 🙌:llama:💯  
 
+## My Builds
+
+In contrast (something softer, like conjunction) with the high level pipeline information on the Pipelines page, the My Builds dropdown menu provides a personalised list of your most recently triggered builds.   
+
+### Navigation item
+
+Living in the navigation bar, the My Builds menu is directly accessible from every page in the Buildkite web application. The count that appears in the My Builds navigation bar item tells you how many of your builds are running or waiting to run.
+
+Screenshot of the nav item
+<%= image "settings.png", width: 728/2, height: 440/2, alt: "Screenshot of the My Builds navigation bar dropdown" %>
+
+### Dropdown
+
+Clicking on the My Builds dropdown menu will display a list of your five most recently created builds.
+This menu displays four key pieces of information about each build: 
+	1. a status icon that indicates passed, failed, and running build states 
+	2. the name of the build or PR
+	3. the repository it was triggered on
+	4. how long ago the build was created
+The `More Builds` button at the bottom of the dropdown menu takes you to a list of all of the builds you've triggered in Buildkite. 
+
+Screenshot of the expanded dropdown
+<%= image "settings.png", width: 728/2, height: 440/2, alt: "Screenshot of the My Builds navigation bar dropdown" %>
+
+### Past builds
+
+The My Builds page lists all of your past builds in reverse chronological order, as well as any that are currently running or waiting for agents. You can filter these to see only 'Running' or 'Scheduled' builds using the buttons in the top right corner. As you scroll down on this page, more builds from your Buildkite history will be loaded until they are all visible.   
```

Comment:
>  As you scroll down on this page, more builds from your Buildkite history will be loaded until they are all visible

We don't do "infinite loading scrolling thingy" so there's a "Next Page" button I think at the bottom of the page. Probably don't need to mention this though!